operator/pkg/ciliumidentity/controller.go

```diff
@@ -245,6 +258,11 @@ func (c *Controller) processNextItem() bool {
 		c.logger.Error("Dropping item from resource queue, exceeded maxRetries", logfields.Key, qItem.Key().String(), "maxRetries", maxProcessRetries, logfields.Error, err)
 	}
 
+	enqueueTime, exists := c.enqueueTimeTracker.GetAndReset(qItem.Key().String())
```

Comment:
What do you think about logging an error here if the item is not tracked?
I assume it's not expected to happen, but if we're checking if it exists, it means it might.

Reply:
Added a warn log.